ush/python/pygfs/jedi/jedi.py

```diff
@@ -188,14 +191,10 @@ def get_obs_dict(self, task_config: AttrDict) -> Dict[str, Any]:
         return obs_dict
 
     @logit(logger)
-    def get_bias_dict(self, task_config: AttrDict) -> Dict[str, Any]:
+    def get_bias(self, task_config: AttrDict, bias_file) -> Dict[str, Any]:
```

Comment:
This is no longer returning a `Dict`
```suggestion
    def get_bias(self, task_config: AttrDict, bias_file) -> None:
```
Also, this is no longer a `get` method.  It is `staging` or `copying` the bias correction files.
The method should be renamed as `stage_bias` or `copy_bias`.

Reply:
@WalterKolczynski-NOAA asked me to change this back to a `get` method.  It now returns a dictionary of files to copy.

Reply:
@aerorahul , is the revised `jedi.py` acceptable?

Reply:
Apologies @RussTreadon-NOAA .
Some of my comments were from a review I had started last week and are now stale/obsolete.  I'll resolve those